src/openapi/lab.yml

```diff
@@ -369,26 +376,57 @@ model:
           - readOnly: true
       name:
         type: string
-        pattern: '^\w+$'
+        maxLength: 64
       description:
         type: string
       enabled:
         type: boolean
-
-  Bundle:
-    type: object
-    properties:
-      id:
-        allOf:
-          - $ref: 'common.yml#/model/ID'
-          - readOnly: true
-      name:
-        type: string
-        pattern: '^\w+$'
-      description:
+      tower_template_name:
         type: string
-      enabled:
-        type: boolean
+        maxLength: 128
+      parameters:
```

Comment:
I'm preparing for my UI work. Could you provide an example on when we would need to specify parameters?

Reply:
It is basically Tower survey spec.